workshops/_template/facilitation-guide.md

```diff
@@ -0,0 +1,28 @@
+# Facilitator's Guide
+## <Workshop Name>
+
+<workshop description>
+
+Duration: XX hours
+Date Updated: XXXX-XX-XX
+
+### Goal
+
+## Materials
+
+Materials needed for the workshop include:
+* 
+*
+*
+
+The repository is structured as follows:
+* `<folder-name>/`
+* `<folder-name>/`
+* `<folder-name>/` with [direct link](#) to important resources
```

Comment:
Oh this is an example of a line. I didn't understand why the structure a sentence that follows.